_episodes/14-checkout.md

```diff
@@ -0,0 +1,315 @@
+
+The teaching demonstration serves several purposes. First and foremost, it is an opportunity to **both give and receive feedback** on
+a more polished presentation after you have had some time to reflect on your lesson and what you have learned in this workshop. In 
+addition, it is an opportunity to **spend some time preparing a lesson** well in advance of your first workshop. This will ease your
+preparations when the date of that workshop approaches. Finally, it is an opportunity to receive **feedback from a Carpentries 
+Instructor Trainer** (like your Trainers at today's event), who will help make sure you are prepared for a strong start.
+
+### Not a High Stakes Test
+It can be nerve-wracking to perform in front of others! This is true when you teach, too. As much as possible, try to relax and
+enjoy this friendly opportunity for feedback and connection with others in our community. Remember to embrace your mistakes!
+
+In some cases, an Instructor Trainer may ask you to repeat your teaching demonstration another time prior to certification. If this
+happens, they will be very clear about exactly what they would like you to change on your second appearance, and why. These requests
+are made with your interests in mind -- we want you to be prepared for success when you set out to teach! 
+
+Trainers draw from their own experience and training when they evaluate teaching demonstrations, but we do provide a 
+[rubric][demos-rubric] for their reference. This can be a helpful resource as you prepare. 
+
+### 1 Lesson = Several Episodes: preparing to teach a lesson.
+For your teaching demonstration, you will prepare to teach **a complete lesson** from one of the
+Carpentries lesson programs. Please refer to the lists of [Data Carpentry lessons]({{ site.dc_site }}/lessons/), [Library Carpentry lessons]({{ site.lc_site }}/lessons/), and [Software Carpentry lessons]({{ site.swc_site }}/lessons/) on the websites. 
+
+A "lesson" corresponds to a single line in the lesson table and a single repository in GitHub.
+(An example of a lesson: [R for Reproducible Scientific Analysis][r-gapminder].
+Some lessons have supplementary modules.
+You do not need to be prepared to teach the supplementary modules for your teaching demonstration.
+
+**The Trainer hosting the teaching demonstration session may choose any episode of the lesson** from this [list of suggested episodes][start-points] for you to teach. (An 
+example for an episode could be: [Data Structures][r-gapminder-episode]). Because we are particularly interested in your use of participatory live coding (or 
+participatory work with spreadsheets etc.), you will not be asked to teach episodes that lack participatory content or that require files or other output from previous episodes.
+other complications to set up.
+
+**If your chosen lesson does not have any suggested episodes for start points listed or is not on the list, please consider choosing a different lesson.** 
```

Comment:
```suggestion
**If your chosen lesson does not have any [suggested episodes for start points][start-points] listed or is not on the list, please consider choosing a different lesson.** 
```

Adding the link again here, as I think it will reduce confusion amongst trainees.